kubernetes/configmap-dashboard.yaml

```diff
@@ -87,6 +87,7 @@ data:
   HAPI_EOS_EXCHANGE_RATE_API: '${HAPI_EOS_EXCHANGE_RATE_API}'
   HAPI_COINGECKO_API_TOKEN_ID: '${HAPI_COINGECKO_API_TOKEN_ID}'
   HAPI_REWARDS_TOKEN: '${HAPI_REWARDS_TOKEN}'
+  HAPI_CREATE_ACCOUNT_ACTION_NAME: '${HAPI_CREATE_ACCOUNT_ACTION_NAME}'
```

Comment:
@AngeloCG97 we are missing an env var here also

Reply:
HAPI_EOS_FAUCET_ACCOUNT_PASSWORD ?

Reply:
Yes, it is
![image](https://user-images.githubusercontent.com/51149817/166971402-07c2e6c1-5d26-4b48-88a2-d2ed86d7e553.png)

Reply:
![image](https://user-images.githubusercontent.com/5632966/166973758-a1064fb7-dd3b-4b87-9360-8646614c6376.png)
 Its in the github workflow but I  dont see it in the config map